code/lib/channel-websocket/src/index.ts

```diff
@@ -67,7 +69,9 @@ export class WebsocketTransport {
     };
     this.socket.onmessage = ({ data }) => {
       const event = typeof data === 'string' && isJSON(data) ? parse(data) : data;
-      this.handler(event);
+      if (this.handler) {
+        this.handler(event);
+      }
```

Comment:
I think we expect a handler to be set here. Could you do a runtime assertion with `tiny-invariant`?
```ts
invariant(this.handler, "WebsocketTransport handler should be set");
```